src/app/Animal/index.ts

```diff
@@ -0,0 +1,15 @@
+import { Organism } from '../Organism';
+
+export abstract class Animal extends Organism {
+  constructor(
+    imageSrc: string,
+    initiative: number,
+    readonly strength: number,
```

Comment:
I'm not sure about mixing `readonly` with regular `constructor` properties. If you think that's OK, I'm OK with that as well.